### ISSUE TITLE
Failed at the strong-arc@1.8.8 preinstall script 'node .sl-blip.js'

### ISSUE THREAD

**Issue by rikantro** [strongloop/loopback#2825]
I try to install strongloop globally but the next error appears

```
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: modern-syslog@^1.1.2 (node_modules\strongloop\n
):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for modern-syslog@1.1.4: wan
:"any"} (current: {"os":"win32","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: modern-syslog@^1.x (node_modules\strongloop\nod
or\node_modules\modern-syslog):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for modern-syslog@1.1.4: wan
:"any"} (current: {"os":"win32","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: sqlite3@3.1.5 (node_modules\strongloop\node_mod
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: sqlite3@3.1.5 preinstall: `npm install node-pre
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: Exit status 4294963214
npm ERR! Windows_NT 10.0.14393
npm ERR! argv "C:\Program Files\nodejs\node.exe" "C:\Users\TCP4\AppData\Roaming\npm\no
-cli.js" "install" "-g" "strongloop"
npm ERR! node v6.7.0
npm ERR! npm v3.10.8
npm ERR! file C:\WINDOWS\system32\cmd.exe
npm ERR! path C:\WINDOWS\system32\cmd.exe
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn C:\WINDOWS\system32\cmd.exe

npm ERR! strong-arc@1.8.8 preinstall: node .sl-blip.js
npm ERR! spawn C:\WINDOWS\system32\cmd.exe ENOENT
npm ERR!
npm ERR! Failed at the strong-arc@1.8.8 preinstall script 'node .sl-blip.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the strong-arc package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR! node .sl-blip.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR! npm bugs strong-arc
npm ERR! Or if that isn't available, you can get their info via:
npm ERR! npm owner ls strong-arc
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR! C:\Users\TCP4\npm-debug.log
npm ERR! code 1
```

Currently I sucessful to install strong-arc module without errors. Only happens with strongloop package

System Information

node 6.7.0
npm 3.10.8
windows 10 x64

**Comment by jjhesk** [comment]
i have experienced the same issue on heroku deploy but it doesnt happen on the localhost.

```
 "async": "https://github.com/caolan/async.git",
    "compression": "https://github.com/expressjs/compression.git",
    "errorhandler": "^1.1.1",
    "loopback": "2.34.1",
    "loopback-context": "https://github.com/strongloop/loopback-context.git",
    "loopback-connector-mysql": "^1.4.7",
    "loopback-boot": "https://github.com/strongloop/loopback-boot.git",
    "loopback-component-explorer": "https://github.com/strongloop/loopback-component-explorer.git",
    "loopback-connector-mongodb": "https://github.com/strongloop/loopback-connector-mongodb.git",
    "loopback-component-storage": "https://github.com/strongloop/loopback-component-storage.git",
    "loopback-connector-redis": "https://github.com/strongloop/loopback-connector-redis.git",
    "loopback-datasource-juggler": "^2.46.1",
    "loopback-filters": "https://github.com/strongloop/loopback-filters.git",
    "strong-wait-till-listening": "https://github.com/strongloop/strong-wait-till-listening.git",
    "gulp-async-func-runner": "https://github.com/Cellarise/gulp-async-func-runner.git",
    "speakeasy": "https://github.com/speakeasyjs/speakeasy.git",
    "imagemagick": "https://github.com/rsms/node-imagemagick.git",
    "aws-sdk": "https://github.com/aws/aws-sdk-js.git",
    "s3": "https://github.com/andrewrk/node-s3-client.git",
    "body-parser": "^1.15.2",
    "express": "4.14.0",
    "req-timeout": "~1.0.2",
    "pug": "2.0.0-beta6",
    "multer-s3": "^2.3.2",
    "mapslice": "^1.2.0",
    "multer": "^1.1.0",
    "gulp-rimraf": "^0.2.0",
    "serve-favicon": "^2.0.1",
    "gm": "^1.22.0",
    "mkdirp-promise": "~3.0.1",
    "lodash": "4.16.1",
    "fs-extra": "^0.30.0",
    "gulp": "^3.9.1"
```

**Comment by rikantro** [comment]
Might be related to https://github.com/strongloop/strong-arc/issues/1733

**Comment by jjhesk** [comment]
how to solve [the problem](https://github.com/strongloop/strong-arc/issues/1733)

**Comment by superkhau** [comment]
Closing this issue as this should've been fixed by strong-arc@1.8.9. Please create a new issue if you are still running into issues.
